general/clients/codec-support.md

```diff
@@ -88,31 +90,54 @@ Subtiles can be a subtle issue for transcoding. Containers have a limited number
 |VobSub<sup>3</sup>|Picture|✅|✅|✅|🔶|
 |MP4TT/TXTT|XML|❌|✅|❌|❌|
 |PGSSUB|Picture|❌|❌|✅|❌|
+|EIA-608/708<sup>4</sup>|Embedded|✅|✅|✅|❌|
 
 <sup>1</sup>MP4 containers can only support one embedded subtitle stream. This does not affect external subtitles.
 
 <sup>2</sup>VTT are supported in an [HLS Stream](https://helpx.adobe.com/adobe-media-server/dev/webvtt-subtitles-captions.html).
 
 <sup>3</sup>DVB-SUB [(SUB + IDX)](https://forum.videohelp.com/threads/261451-Difference-between-SUB-and-IDX-file) is another name for VobSub files.
 
+<sup>4</sup>EIA-608/708 subtitles are embedded in private channels (channel 21) in a mpeg video codec.
+
+### Types of Subtitles
+
+There are many variations of subtitles. Closed, open,  burned-in, forced, SDH, and CC are among the common types of subtitles. The format, such as SubRIP or VobSUB doesn't matter for the type of subtitle.
+
+#### Closed Subtitles
+
+This is the generic name for subtitles that can be turned on or off. This can be Forced, SDH, CC or normal subtitles.
+
+#### Burned-in
+
+Open subtitles, also known as burned-in subtitles, are subtitles that have been permanently placed in the video and cannot be turned off. Open subtitles are the most common type of subtitles, where the subtitles are part of a seperate stream or file and can be toggled on or off.
+
+#### SDH and Closed Captioning
+
+SDH and CC are subtitles for the Deaf and Hard of Hearing. They include extra content such as background noises. SDH and CC are not defined by a specific type of subtitle, just by their intent. If using an OTA Tuner and DVR, the subtitles will be [embedded](https://evertz.com/resources/eia_608_708_cc.pdf) into the video and transcoding them before extracting the subtitles will destroy the subtitles.
```

Comment:
You should probably note the difference between SDH and CC is mainly cosmetic, one being shown on a black band and the other showing up as a regular set of subtitles.

Reply:
SD are enhanced kinda like ASS subs, so they're a little more advanced than just CC. Are black bars a requirement for cc? I figured that was based upon the tv